### PR TITLE
Table on page has a set width somewhere and causing a horizontal scro…

### PR DESCRIPTION
**Pull request by JoshGer**
…llbar

The main table on this page is not displaying correctly. The content in the first column is too big for the table width and so it is causing a horizontal scroll bar on the bottom of the table. This is not a very good experience for customers to view the page content.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@JoshGer : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reporting-services/report-server-web-service/methods/report-server-namespace-management-methods.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ms.author: maggies
 |<xref:ReportService2010.ReportingService2010.CancelJob%2A>|Cancels execution of a job.|  
 |<xref:ReportService2010.ReportingService2010.CreateFolder%2A>|Adds a folder to the report server database or SharePoint library.|  
 |<xref:ReportService2010.ReportingService2010.CreateCatalogItem%2A>|Adds a new item to a report server database or SharePoint library. This method applies to the **Report**, **Model**, **Dataset**, **Component**, **Resource**, and **DataSource** item types.|  
-|M:ReportService2010.ReportingService2010.CreateReportEditSession(System.String,System.String,System.Byte[],ReportService2010.Warning[]@)|Creates a new report edit session.|  
+|M:ReportService2010.ReportingService2010.CreateReportEditSession(System.String,<br/>System.String,System.Byte[],ReportService2010.Warning[]@)|Creates a new report edit session.|
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-server-web-service/methods/report-server-namespace-management-methods.md]:
```suggestion
|M:ReportService2010.ReportingService2010.CreateReportEditSession(System.String, System.String,System.Byte[],ReportService2010.Warning[]@)|Creates a new report edit session.|  
```

Comment (maggiesMSFT) [reply docs/reporting-services/report-server-web-service/methods/report-server-namespace-management-methods.md]: I think just a space will allow it to break on its own.

**Comment by PRMerger13** [comment]
@rothja : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rothja** [comment]
@JoshGer I added the space that Maggie thinks will fix this. I'm going to merge these changes in. If it doesn't look right when published in a few days, please reopen or create a new pull request to suggest your original change. @maggiesMSFT FYI. Thank you!

**Pull request merged**